doc/getting-started.md

```diff
@@ -21,59 +19,72 @@ deployment "istio-mixer" created
 ## Connecting microservices with Istio
 
 This guide shows how to set up Istio and manipulate the proxy mesh to achieve useful behavior.
-In this example we use two microservices:
+In this example we use *CouchDB* and *NGINX* as and demonstrate how to manipulate them:
+
+* CouchDB is the official CouchDB Docker image
+* NGINX is the offical NGINX Docker image.  We will configure it to redirect all calls to CouchDB
 
-* microservice "hello" returns a constant bit of JSON.
-* microservice "frontend" calls microservice "hello"
+First, let's start CouchDB and have it join the Istio service mesh
 
-The communicating microservices are examples from Kubernetes'
-[Connecting a Front End to a Back End Using a Service](https://kubernetes.io/docs/tutorials/connecting-apps/connecting-frontend-backend/) tutorial.
+```
+# Start the official Docker CouchDB image the container in a Kubernetes Pod.  5984 is the CouchDB port.
+kubectl run couchdb --image=docker.io/library/couchdb --port 5984
+```
 
-First we will start the microservices using the Istio pattern of a
-proxy within frontend's pod. The `istioctl kube-inject` command
-injects the istio runtime proxy into kubernetes resource files. It is
-documented [here](istioctl.md#kube-inject).
+The _kubectl get deployment_ subcommand simply creates a machine readable description of a service.  The _istioctl kube-inject_ subcommand creates a machine readable description of a service including Istio components.  By piping the commands together and sending the output to _kubectl apply_ we add Istio capabilities to our service.
 
```

Comment:
Originally I wrote this to set up NGINX and CouchDB without Istio, then I added Istio proxies with _kube-inject_ and had so many problems.  For me, without an in-depth knowledge of Kubernetes (but with strong skills in OpenStack and Cloud Foundry) it seemed to greatly add to the complexity to set up YAML templates for Kubernetes.  Can we not perhaps have several introductions?  After this one is approved we could do one for a more typical Kubernetes-style setup.